[DOCU-1621][DOCU-1638] Konnect: Okta IdP integration (#3045)
Kong/docs.konghq.com: app/konnect/org-management/users-and-roles.md

---
title: Manage Users and Roles
no_version: true
---

## Overview

Many organizations have strict security requirements. For example, organizations
need the ability to segregate the duties of an administrator to ensure that a
mistake or malicious act by one administrator doesn’t cause an outage.

> {{site.konnect_product_name}} provides the ability to manage authorization

Reviewer: I there a reason we're using `product_name` here instead of `short_name`?

through inviting users and granting them defined or custom roles, helping you

> secure your administration environment.

Author: Looks like this word doesn't even need to be here, it only makes the sentence more confusing.

{:.note}
> **Note:** If Okta integration is [enabled](/konnect/org-management/okta-idp),
{{site.konnect_short_name}} users and roles become read-only. An organization
admin can view all registered users in {{site.konnect_short_name}}, but cannot

> edit them from the {{site.konnect_short_name}} side. To manage

Reviewer: ```suggestion
edit their roles from the {{site.konnect_short_name}} side. To manage
```

automatically-created users, adjust user permissions through Okta, or
[adjust role mapping](/konnect/org-management/okta-role-mapping).

## Creating a new user

### Invite a new user
1. In {{site.konnect_saas}}, open the ![org icon](/assets/images/icons/konnect/konnect-organization.svg){:.inline .no-image-expand}
 **Organization > Users** page.
2. Click **Invite New User**.
3. Enter the user’s information, then click **Next**.
5. Assign a role or roles for the new user by checking the box next to the role.
    * A user can have more than one role.

>     * For role descriptions, click the information (`i`) icon next to the role,

Reviewer: ```suggestion
    * For role descriptions, hover over the information (`i`) icon next to the role,
```

>     or see the [Role Definitions section](#role-definitions) below.

Reviewer: ```suggestion
    or see the [Role Definitions](#role-definitions) section below.
```

6. Click **Invite User**.

    An email invitation is sent to the user.

### Accept invite and create account
1. Click on the link in the invitation email to set up your account.
2. Create a password.

>     The password must be 8 characters long and contain at least two of the

Reviewer: ```suggestion
    The password must be 8 characters long and contain at least three of the
```

>     following: a lowercase letter, an uppercase letter, or a special

Reviewer: ```suggestion
    following: a lowercase letter, an uppercase letter, a number, or a special
```
